apps/web/src/services/analytics/Analytics.tsx

```diff
@@ -0,0 +1,23 @@
+import { GA_TRACKING_ID, SAFE_APPS_GA_TRACKING_ID } from '@/config/constants'
+import { GoogleAnalytics } from '@next/third-parties/google'
+import { useEffect } from 'react'
+
+const Analytics = () => {
+  useEffect(() => {
+    // This needs to be added once in order for events with send_to: SAFE_APPS_GA_TRACKING_ID to work
+    window.gtag?.('config', SAFE_APPS_GA_TRACKING_ID, { debug_mode: true })
+
+    window.gtag?.('consent', 'default', {
```

Comment:
How does this work wrt cookies? The current code upgrades the cookie consent once it's given via the banner, and also cookies were deleted once the consent is withdrawn.

Reply:
This is the default consent config we set whenever we called `gtmInit` which we did regardless of cookies being set or not so I think the behaviour should still be the same.